src/mesh/mesh.h

```diff
@@ -70,6 +70,23 @@ class BoundarySegment {
     static Input::Type::Record input_type;
 };
 
+class EquivalentMeshMap{
```

Comment:
Rather change to struct EquivalentMeshMap, in order to:
1. make all public by default
2. emphasize that it is just a pair of maps

Reply:
done